service/core/api/src/main/scala/com/lightbend/lagom/internal/api/UriUtils.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.api
+
+import java.net.URI
+
+object UriUtils {
+
+  /**
+   * Generate a comma separated String containing the host and port of passed URIs.
+   */
+  def hostAndPorts(uris: Seq[URI]): String =
+    uris.map(hostAndPort).mkString(",")
+
+  /** Extra host and port of a URI */
+  private def hostAndPort(uri: URI) = s"${uri.getHost}:${uri.getPort}"
```

Comment:
Funny, I was looking for methods in URI but never suspected that `uri.getAuthority` was the right one. 

I will change it and see if which kind of checks we can do